scripts/install/opt/seagate/cortx/motr/conf/examples/motr_prov_conf.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
The keys of this file needs to match the provisioner spec. 
@nitin-seagate  Can you please validate the keys here.